Nos/Service/RelayService.swift

```diff
@@ -36,9 +36,7 @@ final class RelayService: ObservableObject {
 
         self.saveEventsTimer = AsyncTimer(timeInterval: 1, priority: .high) { [weak self] in
             await self?.backgroundContext.perform(schedule: .immediate) {
-                if self?.backgroundContext.hasChanges == true {
-                    try! self?.backgroundContext.save()
-                }
+                try! self?.backgroundContext.saveIfNeeded()
```

Comment:
We shouldn't force try. Even if we don't expect saveIfNeeded to fail in production, maybe in the future it does and we need to track down all force tries. It would be better to print something to the logs and not crash

Reply:
We have force-trys all over the place that we put there to save time, but I agree that it's time to move away from that. I filed https://github.com/planetary-social/nos/issues/253 to fix them all but I'll fix this one now.